code/game/objects/items/weapons/paint.dm

```diff
@@ -1,82 +0,0 @@
-//NEVER USE THIS IT SUX	-PETETHEGOAT
-//THE GOAT WAS RIGHT - RKF
-
-/obj/item/reagent_containers/glass/paint
```

Comment:
document this removal and tell me why exactly this is removed

Reply:
1. They aren't mapped in at all, to my knowledge.

2. Their only purpose was to recolor the detective trenchcoats. Which even then, was terrible, hardcoded dmi nonsense

3. Not referenced anywhere else in code other than here.